### PR TITLE
[DISTROS] Add support for Pop!_OS 

### PR DESCRIPTION
**Pull request by airvue**
## Description

This PR adds ASCII art for the Ubuntu based distro by System76, Pop!_OS.

## Reason and / or context

This PR was made to expand number of distros supported. Previously, archey would display the Ubuntu logo for this Linux distro.

## How has this been tested ?

Environment:
Tested on my local machine (OS: Pop!_OS 20.04 LTS), and a remote Ubuntu(20.04 LTS) server hosted on an Amazon ec2.

archey-pop

Method:
Used the `packaging/build.sh` script to create the .deb package (.rpm was unable to build, so didn't test; required `rpmbuild` exec).

archey-rpm-error

## Types of changes :

- Bug fix (non-breaking change which fixes an issue)
- Typo / style fix (non-breaking change which improves readability)
- [X] New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist :

- \[IF NEEDED\] I have updated the _README.md_ file accordingly ;
- \[IF NEEDED\] I have updated the test cases (which pass) accordingly ;
- [X] My changes looks good ;
- [X] I agree that my code may be modified in the future ;
- [X] My code follows the code style of this project ([PEP8](https://www.python.org/dev/peps/pep-0008/)).

### CODE REVIEW

File: archey/distributions.py
Comment (HorlogeSkynet) [inline archey/distributions.py:40]: Hey !
One question here too : do you know when Pop! OS decided to ship its own `pop` ID as we can see (here)(https://github.com/KittyKatt/screenFetch/issues/675#issuecomment-632126301) ?
One more proposal : should we provide a "manual" detection procedure for previous (?) Pop! OS releases which were identified as Ubuntu (as your screenshot tells) ?

Thanks again.
Bye 👋

Comment (airvue) [reply archey/distributions.py:40]: I am unsure which version the id was added, but I can try to locate the iso to test previous 18.* & 19.* versions (no longer supported by System76). Perhaps, `cat /etc/*-release` on older releases gives a similar out pictured below:
cat-etc-release
And if the `pop` id is unavailable on older releases, then a workaround could be parsing the `PRETTY_NAME` attribute for `Pop*`.

Additionally, I added an alt Pop!_OS logo by the name `popos`, and fixed a minor color mistake on the other logo. Plan to merge updates soon.
archey-local-alt

Comment (HorlogeSkynet) [reply archey/distributions.py:40]: Thanks for your quick responses. See my points below :

1. I'm okay about implementing a special case for this distribution, expect a commit here soon. Would you be able to try it out ?

2. About alternative logos, unless Neofetch for instance, Archey does not support this mechanism to keep things simple. I'd preferred you only propose the first (fixed) one here. Distribution identifiers should be unique (and match `os-release` if possible). So I guess we should keep `pop` as our candidate.

Bye, 👋

Comment (HorlogeSkynet) [reply archey/distributions.py:40]: Oh sorry I've just understood the screenshot you added in the OP had been taken on your Ubuntu.
No need for a special detection case then (unless previous releases of this OS didn't ship `pop` as `ID` ? Will you be able to easily check this ? I couldn't find Pop's base-files/skeleton/rootfs to verify myself...).
From my PoV, we're good here once your logo fix lands there.

Bye, sorry again for the noises 🙇

Comment (airvue) [reply archey/distributions.py:40]: Logo color changes have been made. All good to go for **Pop!_OS 20.04 LTS**. And okay, I did not include the alternate logo.

archey-local-final

However, checked the **18.04** and **19.10** versions using `cat /etc/*-release` command and they do not include a unique `pop` id, so a special detection case will be needed.

**18.04**
cat-etc-pop18

**19.10**
cat-etc-pop19

Comment (HorlogeSkynet) [reply archey/distributions.py:40]: What a coincidence then 😂
See my patch below, I'm waiting for your feedback before merging this.

Thanks for your logo fix 👌
Bye 👋

**Review by HorlogeSkynet — APPROVED** [review]

**Review by airvue — COMMENTED** [review]
Test case for older Pop!_OS versions looks good :+1:

**Pull request merged**
